elisa/test/styleMock.ts

```diff
@@ -0,0 +1,3 @@
+
+
+export {};
```

Comment:
Is this file necessary?

Reply:
Not at this time, but I decided to keep it in to be consistent with other module jest setups.